test/prog/dftb+/tests

```diff
@@ -40,6 +40,11 @@ helical/10-0Ctube_ELPA                 #? WITH_ELSI and MPI_PROCS <= 8 and MPI_P
 helical/C6H6_stack_ELPA                #? WITH_ELSI and MPI_PROCS == 1 and OMP_THREADS == 1
 helical/10-0Ctube_PEXSI                #? WITH_ELSI and MPI_PROCS <= 8 and MPI_PROCS % 4 == 0 and OMP_THREADS == 1
 
+dispersion/2C6H6_MBD                   #? WITH_MBD and MPI_PROCS <= 4
+#!dispersion/CO2_bulk_TS                 #? WITH_MBD and MPI_PROCS <= 4
+#!dispersion/2C6H6_TS                    #? WITH_MBD and MPI_PROCS <= 4
```

Comment:
once the merge conflicts are resolved, the Libmbd submodule can be updated to the latest version, which has numerical gradients for TS, which should make these two tests functional